src/core/reducers/addons.js

```diff
@@ -194,7 +194,7 @@ export function createInternalAddon(apiAddon: ExternalAddonType): AddonType {
     // TODO: remove this if possible. This is used by core/installAddon
     // and DiscoPane components which do camel case conversions for
     // some historic reason.
```

Comment:
@SIMRAN1, thanks for the PR,  I think you should remove the comments too :)

Reply:
Thanks @championshuttler !